post_installation_configuration/network-configuration.adoc

```diff
@@ -111,6 +111,8 @@ The {product-title} HAProxy router scales to optimize performance.
 include::modules/baseline-router-performance.adoc[leveloffset=+2]
 
 include::modules/router-performance-optimizations.adoc[leveloffset=+2]
+include::modules/ingress-liveness-readiness-startup-probes.adoc[leveloffset=+2]
+include::modules/configuring-haproxy-interval.adoc[leveloffset=+2]
```

Comment:
In the other assembly, these fall under the `router-performance-optimizations.adoc` file. If you meant to do the same here, should be: 
```suggestion
include::modules/ingress-liveness-readiness-startup-probes.adoc[leveloffset=+3]
include::modules/configuring-haproxy-interval.adoc[leveloffset=+3]
```

Reply:
Addressed